python/tests/sbp/client/test_driver.py

```diff
@@ -38,41 +42,44 @@ def tcp_server(handler):
   return (ip, port)
 
 def test_tcp_logger():
-  handler = tcp_handler(MsgPrintDep(text='abc').to_binary())
+  handler = tcp_handler(MsgPrintDep(text=b'abc').to_binary())
   ip, port = tcp_server(handler)
   port = "socket://%s:%s" % (ip, port)
   baud = 115200
   t0 = time.time()
   sleep = 0.1
-  def assert_logger(s):
+  temp_d = {'assert_logger_called': False}  # nonlocal doesn't work in python 2.7
```

Comment:
Maybe something more descriptive name like `assert_ctx`?

Reply:
why not. Doesn't make a big difference in this case since the use is so local but agreed that 1-letter variables names are usually a bad idea